package.json

```diff
@@ -19,43 +19,34 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w ./lib",
+    "lint": "jshint lib/**/*.js test/**/*.js --config .jshintrc",
```

Comment:
I'd suggest using:

```
     "lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc", 
```

Due to:

1. We have two `.jshintrc` to have the possibility of using a different style for regular code and test (although maybe currently we are not using that and both `.jshintrc` are basically the same).
2. It seems the `lib/**/*.js` glob doesn't cover the content on `lib/` directory itself. It seems that `lib/` works in a recursive way and doesn't have that problem. It is easy to check: remove a `;` in `lib/constant.js` and you will see that current approach doesn't detect while my above proposal does.